packages/apps/plugins/plugin-deck/src/DeckPlugin.tsx

```diff
@@ -323,6 +324,14 @@ export const DeckPlugin = ({
             }
 
             case NavigationAction.OPEN: {
+              if (intent.data?.object) {
+                // forward to `EXPOSE`
+                void intentPlugin?.provides.intent.dispatch({
+                  action: NavigationAction.EXPOSE,
+                  data: { id: fullyQualifiedId(intent.data.object) },
+                });
+              }
```

Comment:
Add to list of returned intents instead?

Reply:
Ah smart, yes.